xml/decoders/trix/mLD3_mSD3_enum_triggers.xml

```diff
@@ -111,6 +111,102 @@
     <choice xml:lang="fr">F15</choice>
     <choice xml:lang="cs">F15</choice>
   </enumChoice>
+  <enumChoice>
+    <choice>F16</choice>
+    <choice xml:lang="de">F15</choice>
+    <choice xml:lang="fr">F15</choice>
+    <choice xml:lang="cs">F15</choice>
```

Comment:
This doesn't look right. Shouldn't it be F16 for all the languages? Same applies for the other functions as well.

Reply:
@patnaik-g  Could you take a look at this please?  Looks like all the new enum choices reference F15 instead of e.g. F16, F17, etc

Reply:
Sorry for that. I had them changed in a different version of the file. Fixed. New file attached.

[mLD3_mSD3_enum_triggers.txt](https://github.com/JMRI/JMRI/files/13910183/mLD3_mSD3_enum_triggers.txt)

Reply:
I've merged that new file into this branch. @danielb987 we're ready for another review, thanks.